pyproject.toml

```diff
@@ -1,27 +1,70 @@
 [build-system]
-requires = ["setuptools", "setuptools_scm", "wheel"]
-build-backend = 'setuptools.build_meta'
+requires = ["sphinx-theme-builder >= 0.2.0a7"]
+build-backend = "sphinx_theme_builder"
```

Comment:
I really hope we don't need to do this.

Unless we need to actually be running webpack because we have custom build js or scss or whatever we shouldn't need this and we can just be a normal python package using setuptools or flit etc.

Reply:
I did this for fun.